app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: ''
```

Comment:
Have a default configurable address?  Is this line necessary?

Reply:
Seems related to L4-5 in `app/mailers/assignment_mailer.rb`

Reply:
I will address this in the next PR which will include progress bar, assignment timeout and email functionalites.